app/js/handlebarsScoreTable.js

```diff
@@ -140,14 +140,14 @@ function produceTable (HBTemplate, scoresDataObject) {
         }
     })
 
-    let template = Handlebars.compile(HBTemplate)
-    let score_list = document.querySelector(".score_list")
-    score_list.innerHTML = ""
-    let html = template(scoresDataObject)
-    score_list.innerHTML += html
+    let template = Handlebars.compile(HBTemplate);
+    let score_list = document.querySelector(".tableList");
```

Comment:
Nothing major, just semantics - it threw us a little that the var name is still score_list but you're selecting the table list. 
Is it worth changing score_list to something more semantic?